client/components/icons.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+class Icons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentIcon: 1
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    this.setState({ currentIcon: Number(event.target.id) });
+  }
+
+  render() {
+    return (
+      <div className='row hide-desktop absolute-bottom-icons margin-0 text-align-center'>
+        <div className='column-fifth'>
+          <a href='#marketplace'>
+            <i onClick={this.handleClick} className={this.state.currentIcon === 1 ? 'fas fa-store-alt fa-2x orange' : 'fas fa-store-alt fa-2x '} id='1'></i>
```

Comment:
Here you can check `this.props.path === 'marketplace' ?` so your icon styling stays in sync with your hash route.